tests/cypress/integration/pdv_hoc_site_spec.js

```diff
@@ -29,7 +29,31 @@ describe('Test H.O.C.', () => {
       })
   })
 
-  it('OIE release notes (en-US)', () => {
+  it('Classic/V1 release notes production page tabs work as expected', () => {
+    cy.visit('en-us/Content/Topics/ReleaseNotes/production.htm')
+    // The relnotes production page has four tabs
+    cy.get('ul[id="production-tabs"] > li')
+      .should('have.length', 4)
+      // Each tab can be activated
+      .each(($li) => {
+        cy.wrap($li).click()
+          .should('have.class', 'is-active')
+      })
+    })
+
+    it('Classic/V1 topics have correct script formatting', () => {
+      cy.visit('en-us/Content/Topics/ReleaseNotes/production.htm')
+      // Most scripts take the 'defer' attr
+      cy.get('head script[src*="MadCapAll.js"')
+        .should('have.attr', 'defer')
+      // The following scripts must have 'defer' attrs removed
+      cy.get('head script[src*="require.min.js"]')
+        .should('not.have.attr', 'defer')
+      cy.get('head script[src*="foundation.6.2.3_custom.js"]')
+        .should('not.have.attr', 'defer')
+    })
+
+  it('OIE release notes (en-US) landing page has three tiles', () => {
```

Comment:
same nit as above "landing page has the correct number of tiles"

Reply:
Ah, yes indeed. Making the comment more generic.